cpa/tester/policies/helpers/contexts/policy.rego

```diff
@@ -0,0 +1,17 @@
+package org
+
+import data.circleci.config
+
+policy_name["context_lists"]
+
+enable_rule["context_allowlist"]
+
+context_allowlist := config.context_allowlist_by_project_id(data.meta.project_list, data.meta.allowlist)
+
+enable_rule["context_blocklist"]
+
+context_blocklist = config.context_blocklist_by_project_id(data.meta.project_list, data.meta.blocklist)
```

Comment:
should we disambiguate `project_list`, ie whether it is project_id_list or project_slug_list?

Reply:
This is internal function definition. We have the context from the name of the function. I don't think it is necessary